code/modules/clothing/head/misc.dm

```diff
@@ -514,3 +514,13 @@
 	icon_state = "JackFrostHat"
 	item_state = "JackFrostHat"
 
+/obj/item/clothing/head/more
+	name = "hat of more hats"
+	desc = "Chop chop!"
+	icon_state = "morehat"
+	item_state = "morehat"
+
+/obj/item/clothing/head/more/attack_self(mob/M)
+	var/list/hat_list = subtypesof(/obj/item/clothing/head)
+	var/hat = pick(hat_list)
+ 	new hat(get_turf(M))

```

Comment:
Can this summon those temporary holders that hold drones and pAIs when they're being used as hats? Can this summon ERT helmets?

*gasp*
Can this summon a... a... a-a-a CAKEHAT?!